py-polars/requirements-dev.txt

```diff
@@ -23,6 +23,7 @@ cloudpickle
 hypothesis==6.79.4; python_version < '3.8'
 hypothesis==6.80.0; python_version >= '3.8'
 maturin==1.1.0
```

Comment:
shouldn't it be `maturin[patchelf]`?

Reply:
`patchelf` isn't available on Windows, so that unfortunately fails. The syntax below would work, which amounts to the same thing:

```
maturin[patchelf]==1.1.0; platform_system == 'Linux'
maturin==1.1.0; platform_system != 'Linux'
```